dashboard/src/app/factories/create-factory/create-factory.html

```diff
@@ -25,28 +25,20 @@
   <div class="create-factory" ng-hide="createFactoryCtrl.isLoading">
     <ng-form name="factoryNameForm">
       <!-- Name -->
-      <che-label-container che-label-name="Name">
-        <div layout="column" class="create-factory-input"
-             ng-init="createFactoryCtrl.setForm(factoryNameForm)">
-          <che-input-box che-form="factoryNameForm"
-                         che-name="name"
-                         che-place-holder="Name of the factory"
-                         aria-label="Name of the factory"
-                         ng-model="createFactoryCtrl.name"
-                         ng-trim
-                         ng-minlength="3"
-                         ng-maxlength="20"
-                         ng-pattern="/^[ A-Za-z0-9_\-\.]+$/">
-            <div ng-message="required">A name is required.</div>
-            <div ng-message="pattern">Factory name may contain digits, latin letters, spaces, _ , . , - and should start
-              only
-              with digits, latin letters or underscores
-            </div>
-            <div ng-message="minlength">The name has to be more than 3 characters long.</div>
-            <div ng-message="maxlength">The name has to be less than 20 characters long.</div>
-          </che-input-box>
-        </div>
-      </che-label-container>
+      <che-input-box che-label-name="Name"
+                     che-form="factoryNameForm"
+                     che-name="name"
+                     che-place-holder="Name of the factory"
+                     aria-label="Name of the factory"
+                     ng-model="createFactoryCtrl.name"
+                     ng-trim
+                     ng-minlength="3"
+                     ng-maxlength="20"
+                     ng-pattern="/^[ A-Za-z0-9_\-\.]+$/">
+        <div ng-message="pattern">Factory name may contain digits, latin letters, spaces, _ , . , - and should start only with digits, latin letters or underscores</div>
```

Comment:
I removed "ng-message =required" because we have another solution(red placeholder with an asterisk) in mockups;
![selection_010](https://user-images.githubusercontent.com/6310786/27183115-33bf5e56-51e6-11e7-8329-960c018813ba.png)